var/spack/repos/builtin/packages/cudnn/package.py

```diff
@@ -366,7 +373,7 @@ def url_for_version(self, version):
                 .replace("darwin", "osx")
                 .replace("aarch64", "aarch64sbsa")
             )
-        elif version < Version("8.8.0"):
+        else:
```

Comment:
I just switched to sbsa for all versions above `8.3.1`. Beside `8.9.0.131-12` the files are identical, and sbsa is in most cases the correct architecture as aarch64 is ment for the NVIDIA Jetson lineup.